Basic/APIDiorama/Third Party Notices.md

```diff
@@ -0,0 +1,26 @@
+This package contains third-party software components governed by the license(s) indicated below:
+---------
+
+Component Name: UnitySceneReference
```

Comment:
don't forget to cleanup, this should be removed.

Reply:
Good catch, thanks! Done